core/ir/x86/decode.c

```diff
@@ -2410,6 +2436,22 @@ decode_get_tuple_type_input_size(const instr_info_t *info, decode_info_t *di)
         di->input_size = OPSZ_NA;
 }
 
+/* TODO i#6238: Not all opcodes have been reviewed.
+ * In case an opcode has not been reviewed,
+ * the default category assigned to it is DR_INSTR_CATEGORY_OTHER.
+ */
+static inline void
+decode_category(instr_t *instr)
+{
+    if (instr) {
+        if (op_instr[instr->opcode])
+            instr_set_category(instr, op_instr[instr->opcode]->category);
```

Comment:
But what about the other comment thread: we're going to be missing a lot of _LOAD and _STORE and we need to add them here for instr_reads_memory() or instr_writes_memory().  That seems simple enough to just add here (a couple of lines) instead of adding a TODO for later?

Reply:
agree